spec/defines/admin_spec.rb

```diff
@@ -5,14 +5,14 @@
     context "on #{os}" do
       let(:facts) { os_facts }
       let(:title) { 'help' }
-      let(:pre_condition) do
-        <<-PUPPET
-          include pulpcore
-        PUPPET
-      end
 
-      context 'with a fixed pulp_settings' do
-        let(:params) { { pulp_settings: '/etc/pulpcore/settings.py' } }
+      context 'with a fixed pulp_settings and static_root' do
+        let(:params) do
+          super().merge(
```

Comment:
There is no super so just define it as a regular hash

Reply:
:+1:, thanks